pkg/event/event_suite_test.go

```diff
@@ -0,0 +1,143 @@
+package event_test
+
+import (
+	"fmt"
+	"testing"
+
+	k8sV1 "k8s.io/api/core/v1"
+
+	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/submariner-io/submariner/pkg/event"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestEventPackage(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Event suite")
+}
+
+type TestEvent struct {
+	Name      string
+	Handler   string
+	Parameter interface{}
+}
+
+type TestHandler struct {
+	event.HandlerBase
+	Name          string
+	NetworkPlugin string
+	FailOnEvent   error
+	Events        chan TestEvent
+	Initialized   bool
+}
+
+var allTestEvents chan TestEvent = make(chan TestEvent, 1000)
+
+func cleanupTestEvents() {
+	allTestEvents = make(chan TestEvent, 1000)
+}
+
+func NewTestHandler(name, networkPlugin string) *TestHandler {
+	return &TestHandler{
+		Name:          name,
+		NetworkPlugin: networkPlugin,
+		Events:        make(chan TestEvent, 100),
+		FailOnEvent:   fmt.Errorf("Test handler not initialized"),
```

Comment:
We can remove the initialization here since we're no using it.

Reply:
right.